JsonClient/JsonClient/AbstractManageObject.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using JsonClient.Attributes;
+namespace JsonClient
+{
+  public abstract class AbstractManageObject
+  {
+    public override string ToString()
+    {
+      string ObjectString= "";
```

Comment:
Please take a look into this document of [code conventions](https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/)